shared/css/font.scss

```diff
@@ -6,9 +6,13 @@
 // Try to avoid use of inline styles if you're creating new component).
 // Keeping that in mind -
 
-$metropolis-font: 'Metropolis', sans-serif;
+$metropolis-font: 'Metropolis';
+$noto-sans-fonts: 'Noto Sans', 'Noto Sans Arabic', 'Noto Sans Armenian', 'Noto Sans Bengali', 'Noto Sans SC',
```

Comment:
Does this font family list include all of the languages supported by Code.org?

Reply:
Correct, this list covers all the supported languages by Code.org.
I also added comments about which font supports which language: https://github.com/code-dot-org/code-dot-org/blob/128bdd7b39ddcc8eefec4257cf3c0412f55a521a/shared/css/fonts/noto-sans.scss#L16-L21